src/app.js

```diff
@@ -27,6 +29,56 @@ const adminBro = new AdminBro({
     rootPath: '/admin',
 })
 
+// IEX Api
+const quote = async (sym) => {
```

Comment:
Change this method so that quote takes the whole stock (with the market field populated) as the argument. This way, you can access its name inside the function anyway but you won't have to do Market.findOne() and Stock.findOne(). You can modify that stock directly and then call stock.save(). This will save some database calls and will make the process faster.

Reply:
The Market.findOne() function has been called to check whether the Market Index instance to which stock belongs is present in the databse or not. The quoteData variable will have the market field anyways so I don't see any point of having it in the models unless you plan to have different exchangeRate, priceRateChangeFactor and I'll have to use it anyway to get access to the id of that market model to relate it with the stocks model.
The Stock.findOne() has been used to see if the stock searched already exists in the database or not, if it exists then the initialStockPrice is not changed else that field is populated with the latest price, as users are not having the flexibility to search for any stock randomly and all the stocks must be already present in the database this could have been avoided and I'll remove it but I think that won't make much of a difference as this is called once in a hour for each share, so is not that expensive.